app/src/main/res/layout/fragment_custom_selector.xml

```diff
@@ -16,14 +16,13 @@
     android:id="@+id/empty_text"
     android:layout_height="wrap_content"
     android:layout_width="wrap_content"
-    android:textSize="@dimen/normal_text"
+    android:textSize="20sp"
     android:padding="@dimen/standard_gap"
-    android:text="@string/custom_selector_empty_text"
-    android:visibility="gone"
```

Comment:
Make visibility as `gone` in initial state and make it visible if there are no images. It is visible even when the loader is active.